cmd/lint.go

```diff
@@ -12,11 +12,16 @@ func setupLintCommand() *cobra.Command {
 		Short: "Lint the integration",
 		Long:  "Use lint command to lint the integration files.",
 		RunE:  lintCommandAction,
+		SilenceUsage: true,
```

Comment:
I wonder if there is global switch to fix these cases. Otherwise we need to mute usage for every subcommand.

Reply:
It might be possible to set SilenceUsage at the root command level, but I'm 100% sure based on this thread: https://github.com/spf13/cobra/issues/340. I will try it out and update this PR accordingly.

Reply:
Fixed in 5ffd82d.